plugins/importexport/native/NativeImportExportDeployment.inc.php

```diff
@@ -66,10 +66,11 @@ function getSchemaFilename() {
 	/**
 	 * Set the import/export issue.
 	 * @param $issue Issue
+	 * @param $addProcessedObject boolean Should this issue be identified as a newly processed object?
 	 */
-	function setIssue($issue) {
+	function setIssue($issue, $addProcessedObject = true) {
 		$this->_issue = $issue;
-		if ($issue) $this->addProcessedObjectId(ASSOC_TYPE_ISSUE, $issue->getId());
+		if ($issue && $addProcessedObject) $this->addProcessedObjectId(ASSOC_TYPE_ISSUE, $issue->getId());
```

Comment:
Really, it is the `NativeXmlIssueFilter` which should act on its `$deployment` object to call `addProcessedObjectId()`, similar to the way we handle sections there.  That's where we really decide if we're adding a new Issue.